src/ArchiveAdmin.php

```diff
@@ -313,10 +313,10 @@ public function Breadcrumbs($unlinked = false)
      */
     public function getManagedModels()
     {
-        $models = $this->getVersionedModels();
+        $archivedModels = $this->getVersionedModels();
 
         // Normalize models to have their model class in array key and all names as the value are uppercased
-        foreach ($models as $k => $v) {
+        foreach ($archivedModels as $k => $v) {
             $archivedModels[$v] = array('title' => ucfirst(singleton($v)->i18n_plural_name() ?? ''));
             unset($archivedModels[$k]);
         }
```

Comment:
We're now modifying the array we're looping over - could this introduce problems such as accidentally skipping items, or running over an item a second time (if `$archivedModels[$v]` gets added to the end of the array because it wasn't there before), or ending up with both a `$archivedModels[$k]` _and_ `$archivedModels[$v]` entry where before we only had `$archivedModels[$v]`?

Reply:
@alessandromarotta Can you please address the concerns in my above comment?